website/docs/reference/warehouse-profiles/vertica-profile.md

```diff
@@ -8,12 +16,24 @@ Some core functionality may be limited. If you're interested in contributing, ch
 
 :::
 
+
+| Attribute                           | Value                                       |
+| ----------------------------------- | ------------------------------------------- |
+| **Maintained by**                   | <p>{frontMatter.meta.maintained_by}</p>     |
+| **Authors**                         | <p>{frontMatter.meta.authors}</p>           |
+| **Github Repo**                     | <p>{frontMatter.meta.github_repo}</p>       |
+| **Supported dbt Core version**      | <p>{frontMatter.meta.core_version}</p>      |
+| **dbt Cloud support**               | <p>{frontMatter.meta.cloud_support}</p>     |
+| **Supported data platform version** | <p>{frontMatter.meta.supported_version}</p> |
```

Comment:
@JKarlavige any idea why this isn't working? I feel like we're so close!!!
<img width="881" alt="image" src="https://user-images.githubusercontent.com/8158673/191345995-c28cb58e-6efb-4e32-928b-726f454ea3da.png">

Reply:
> @JKarlavige any idea why this isn't working? I feel like we're so close!!!

It looks like the frontmatter tags don't play nice with the markdown tables. There might be a way to make those `{ }` tags work in the markdown table, but i'm not aware of a way.

However, using a regular html table will work, although not as pretty.
![image](https://user-images.githubusercontent.com/13647110/191350860-f57ed12e-a118-4206-ae0a-2d3b8fe1b334.png)

One thing I had to adjust was the `github_repo` link. The markdown link wasn't working from the frontmatter. Here's all the code with the adjustments, which produced the screenshot above:

```
---
meta:
  maintained_by: Community
  authors: Matthew Carter, Andy Regan, Andrew Hedengren
  github_repo: 'https://github.com/mpcarter/dbt-vertica'
  link_text: 'dbt-vertica'
  core_version: 'v0.21.0 and newer'
  cloud_support: Not Supported
  supported_version: Vertica 10.0+
---
<table>
  <thead>
    <tr>
      <th>Attribute</th><th>Value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Maintained by</strong></td>
      <td><p>{frontMatter.meta.maintained_by}</p></td>
    </tr>
    <tr>
      <td><strong>Authors</strong></td>
      <td><p>{frontMatter.meta.authors}</p></td>
    </tr>
    <tr>
      <td><strong>Github Repo</strong></td>
      <td><p><a href={frontMatter.meta.github_repo}>{frontMatter.meta.link_text}</a></p></td>
    </tr>
    <tr>
      <td><strong>Supported dbt Core version</strong></td>
      <td><p>{frontMatter.meta.core_version}</p></td>
    </tr>
      <tr><td><strong>dbt Cloud support</strong></td>
      <td><p>{frontMatter.meta.cloud_support}</p></td>
    </tr>
    <tr>
      <td><strong>Supported data platform version</strong></td>
      <td><p>{frontMatter.meta.supported_version}</p></td>
    </tr>
  </tbody>
</table>
```